addons/web_editor/static/src/scss/web_editor.frontend.scss

```diff
@@ -58,7 +58,7 @@
 }
 
 .o_grid_item_image {
-    > img {
+    > img, .media_iframe_video {
```

Comment:
Why not `>` just like for the images ? It would make the CSS rules more efficient but it will mainly be more consistent.